app/assets/javascripts/channels/maneuver_participants.coffee

```diff
@@ -12,5 +12,8 @@ App.manuever_participants =
         a.text(score)
       else
         cell.text(score)
-      cell.attr('data-text', score).attr('data-score', score)
+      cell.attr('data-text', score).attr('data-score', score).addClass('last-updated')
       $("table.scoreboard").trigger("recalculate")
+      setTimeout ->
+        cell.removeClass('last-updated')
+      , 5000
```

Comment:
Abstract this update code so we can update it easily later

Reply:
Yeah good call my b